lib/install.js

```diff
@@ -156,7 +150,7 @@ function install (gyp, argv, callback) {
   }
 
   function getContentSha(res, callback) {
-    var shasum = crypto.createHash(checksumAlgo)
+    var shasum = crypto.createHash(release.checksumAlgo)
```

Comment:
Should we bother checking if the Algo is available? Everything has SHA265 these days I think?